regression_tests/i356-enum-without-int-helpers/tests/bindings/test.swift

```diff
@@ -0,0 +1,4 @@
+import regression_test_i356
+
+assert(which(arg: true) == Which.yeah)
+assert(which(arg: false) == Which.nah)
```

Comment:
Nit: exclude `Which`. i.e. 

```swift
assert(which(arg: true) == .yeah)
```

Reply:
Also: I note that Swift is using the `UInt32.read` extension methods, so should we change that now? or file an issue?

Reply:
Nice catch. It looks like the swift bindings generate these helpers unconditionally, rather than inspecting the types used in the UDL. Let's file an issue for followup to do conditional generation, and we can count on this new test to catch the problem if we reintroduce it there. (I confirmed that the test would catch it by commenting out the `extension UInt32: Primitive, ViaFfi` in the generated swift bindings, and this swift test does indeed fail with an appropriate compile-time error in that case).

Reply:
I filed https://github.com/mozilla/uniffi-rs/issues/362 for followup.